### ISSUE TITLE
Adding options to appearence-> custom menu items using cmb2

### ISSUE THREAD

**Issue by sandrodz** [CMB2/CMB2#1129]
Is it possible to add options to a menu item?

Similar to how wp-multilang adds those language checkboxes?

<img width="476" alt="screen shot 2018-05-06 at 3 15 50 am" src="https://user-images.githubusercontent.com/8479569/39668328-de51dac0-50db-11e8-93fb-4d050ad9cabd.png">

**Comment by tw2113** [comment]
I do not believe this type of feature/support has been added since that issue was originally closed, so very likely still not something available out of the box.
